package.json

```diff
@@ -22,10 +22,14 @@
   "scripts": {
     "clean": "rm -rf cache out",
     "build": "forge build",
+    "deploy": "forge create Resolver --rpc-url $RPC_URL --private-key $PRIVATE_KEY --constructor-args $EAS_ADDRESS_ARBITRUM",
```

Comment:
You can set this to EAS_ADDRESS. The implementer can choose among all available chains which address to use